### DISCUSSION TITLE
HTTP500 during ADIF export

### DISCUSSION THREAD

**Discussion by HakanKoseoglu** [magicbug/Cloudlog#1862] [General]
Possible regression. I am not sure if I should raise an issue. Can anyone else replicate this?

Code at: commit 8b9d2598ba248e9cff60ba8b26e1f25f43d4833a (HEAD -> master, tag: 2.3, origin/master, origin/HEAD)

Edit: ran it on developer mode, additional log:

_An uncaught Exception was encountered
Type: Error

Message: Call to a member function format() on bool

Filename: /var/www/html/application/models/Adif_data.php

Line Number: 87_

Open ADIF Import/Export, select Export.

Selected logbook, did not click on any other options. Fails with HTTP500 as below, which isn't very helpful. 

172.20.0.1 - - [22/Dec/2022:22:29:24 +0000] "POST /index.php/adif/export_custom HTTP/1.1" 500 295 "https://logbook.koseoglu.org/index.php/adif" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36"

If I enter start/end dates, this executes w/o a problem.

![image](https://user-images.githubusercontent.com/10955987/209236783-ac891341-880e-49eb-990f-12116fa11fa0.png)

**Comment by MikleBas** [comment]
Hey!
I repeated your experiment - I confirm, the same behavior.
This is (probably) due to the fact that your hosting is running on php version 8*
If you transfer your site to version 7.4 (as I had originally), then this problem does not exist. But the site works many times slower, at times.

**Comment by phl0** [comment]
This bug should be fixed by now (in the dev branch).
